osu.Game.Rulesets.Osu/Objects/Slider.cs

```diff
@@ -20,24 +20,33 @@ public class Slider : OsuHitObject, IHasCurve
         /// </summary>
         private const float base_scoring_distance = 100;
 
-        public IHasCurve CurveObject { get; set; }
-
-        public SliderCurve Curve => CurveObject.Curve;
+        public SliderCurve Curve { get; } = new SliderCurve();

```

Comment:
i generally prefer `public readonly SliderCurve Curver = new SliderCurve();` for these cases, what you think?